### PR TITLE
Home producer branch

### PR DESCRIPTION
**Pull request by ViniciusdePSouza**
In this branch, I implemented a useEffect hook to fetch all events from the database. Then, I filtered them into two types of events: the ones that have already passed and the ones that are featured. I saved each of these arrays in two states, which I used to render the data on the screen. As for the styles, I only added the event banner to the project.

### CODE REVIEW

File: itoddy-role/package.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     "@hookform/resolvers": "^3.1.0",
     "@radix-ui/react-tabs": "^1.0.4",
     "axios": "^1.4.0",
+    "dayjs": "^1.11.7",
     "json-server": "^0.17.3",
+    "luxon": "^3.3.0",
```
Comment (wendersonejr) [inline itoddy-role/package.json:21]: Como está utilizando o dayjs, o Luxon pode ser removido, não?

**Review by wendersonejr — APPROVED** [review]
Numa próxima branch, troca as medidas como padding, margim para pixel. Fica mais fácil de entender.

Para gerar as imagens aleatórias, usa essa url aqui: https://picsum.photos/200

**Review by wendersonejr — APPROVED** [review]
Numa próxima branch, troca as medidas como padding, margim para pixel. Fica mais fácil de entender.

Para gerar as imagens aleatórias, usa essa url aqui: https://picsum.photos/200

File: itoddy-role/package.json (hunk 1 above)
Comment (ViniciusdePSouza) [reply itoddy-role/package.json:21]: positivo

**Pull request merged**